.changesets/fix_router-external-extensibility-deprecation.md

```diff
@@ -0,0 +1 @@
+###  We have deprecated the external extensibility and now have updated error log statements to support coprocessor. By [@briannafugate408](https://github.com/briannafugate408) in https://github.com/apollographql/router/pull/6791
```

Comment:
It looks like now everything is in the headline. That is now too long.
Let's make a shorter headline with PR link (make it shorter using a hyperlink).
And keep the longer description and your name in the body.